src/aquarium.py

```diff
@@ -92,6 +93,11 @@ async def aquarium_startup(aquarium_app: FastAPI, aquarium_api: FastAPI):
         gstate.config.options.services.probe_interval, gstate, nodemgr)
     gstate.add_services(services)
 
+    network: NetworkInterfaces = NetworkInterfaces(
```

Comment:
(nit) should this be `networks`?

Reply:
I vote for `network_interfaces` or `network_ifaces`.